theme/templates/pages/group.html

```diff
@@ -381,36 +391,37 @@ <h4><strong>{{ pendingrequest.invitation_to|contact }}</strong>
                     </div>
                 </div>
 
-                {% if profile_user.is_group_owner or profile_user.is_group_editor or profile_user.is_group_viewer %}
-                    <div role="tabpanel" class="tab-pane fade in active" id="resources">
+                <div role="tabpanel" class="tab-pane fade in active" id="resources">
```

Comment:
Show the Resources tab regardless of permissions held by the viewer; resource listing will be restricted further down. Overview tab has been removed.